### DISCUSSION TITLE
Hiding List Item

### DISCUSSION THREAD

**Discussion by CynthiaBlue** [quasarframework/quasar#11961] [CLI - PWA mode]
I have a q-list with items, and i want to show/hide the items depending on a flag in the database.  I'm using a v-for and a v-if which seems to be frowned upon. And when I hide one of the items, a line still appears in this list but it is blank.  I'd like the list not to show anything for that item at all.  

This is my code:
'''

        <q-list separator> 

          <q-item
            v-for="allTheTrials in theTrials"
            :key="allTheTrials.id"
            class="cursor-pointer"
            clickable
          >
          <div class="col-12 rowStyle" v-if="allTheTrials.fb_publishEvent">
            <q-item>
            <q-item-section
              avatar
              @click="viewEntry(allTheTrials.id)"
              clickable
             >
            <q-icon name="las la-dot-circle" />
            </q-item-section>

            <q-item-section
              avatar
              @click="viewEntry(allTheTrials.id)"
              clickable
              >
              <q-item-label :class="{ 'text-strike' : allTheTrials.done }">
                {{ allTheTrials.fb_eventName }} - {{ allTheTrials.fb_nameOfClub }}
              </q-item-label>
            </q-item-section>

            <q-item-section
              @click="viewEntry(allTheTrials.id)"
              clickable>
              <q-item-label>
               {{ allTheTrials.fb_eventStartDate }} - {{ allTheTrials.fb_secretaryBusinessName }}
              </q-item-label>
            </q-item-section>

            <q-item-section side>
              <q-item-label caption>{{ allTheTrials.fb_eventType }}</q-item-label>
            </q-item-section>              
            </q-item>
          </div>
         
          </q-item>
        </q-list>
'''

And this is the results I get.  Is there a way to make that empty line disappear?   Thanks! 

<img width="779" alt="Screen Shot 2022-01-09 at 9 48 50 AM" src="https://user-images.githubusercontent.com/34698014/148692129-c010338e-5102-4234-8c63-fb461178b1c7.png">

a

**Comment by MilosPaunovic** [comment]
Create a computed property with items you want to show.
